addons/skin.estuary/xml/Includes_MediaMenu.xml

```diff
@@ -337,7 +337,7 @@
 			<height>100%</height>
 			<onleft>9000</onleft>
 			<onup>9000</onup>
-			<onright>50</onright>
+			<onright>55</onright>
```

Comment:
This was changed as I couldn't move off the sideblade menu back to the list, I assume because View 50 List is not used for any of the views for Playlist window.

I've done my best to look at Games, PVR, & Addons to check for any adverse affect of changing this and I wasn't able to spot any. Probably needs more extensive testing though.